sunflower-land/sunflower-land
# [FEAT] Marketplace Listing

# Description

List an item for sale in the marketplace:

<img width="659" alt="Screenshot 2024-09-18 at 10 14 40 PM" src="https://github.com/user-attachments/assets/bc897022-2048-46c7-b242-4e1472b5a6ad">

Fixes #issue

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

- Click on "List an item"
- Check for both onchain and offchain items

# Checklist:
